docs/examples/index.rst

```diff
@@ -1,41 +1,74 @@
 Examples
 ========
 
-The examples below highlight key elements of how ownership and control information is built up through the use of statements.
+These fictional examples highlight key elements of how ownership and control information is built up through the use of statements.
 
-You can also find examples of valid BODS JSON files in the standard repository:
+A single direct owner
+---------------------
 
-- Example data (`v0.3 <https://github.com/openownership/data-standard/tree/0.3.0/examples>`__  |  `v0.2 <https://github.com/openownership/data-standard/tree/0.2.0/examples>`__  | `v0.1 <https://github.com/openownership/data-standard/tree/0.1.0/examples>`__ )
-- Test data (`v0.3 <https://github.com/openownership/data-standard/tree/0.3.0/tests/data/entity-statement/valid>`__  |  `v0.2 <https://github.com/openownership/data-standard/tree/0.2.0/tests/data/entity-statement/valid>`__  | `v0.1 <https://github.com/openownership/data-standard/tree/0.1.0/tests/data/entity-statement/valid>`__ )
+Three statements (Entity, Person and Relationship) that describe the 100% beneficial ownership of Profitech Ltd by Jennifer Hewitson-Smith. 
 
-For a visual rendering of these examples, use the `BODS Visualiser <https://www.openownership.org/visualisation/visualisation-tool/>`_.
+.. literalinclude:: ../../examples/bods-package.json
+    :language: json
 
+Tecido
+------
 
-A single direct owner
----------------------
+This example demonstrates a gradual transfer of ownership from a single owner to an employee owned trust. In this jurisdiction a beneficial owner must be declared when they own at least 25% of the company. 
```

Comment:
```suggestion
This example demonstrates a gradual transfer of ownership from a single owner to an employee owned trust. In this fictional example, a beneficial owner must be disclosed when they own at least 25% of the company. The trust arrangement itself is not disclosed.
```

Reply:
have removed 'fictional' because it's stated at the top and is implied